### ISSUE TITLE
App crash at SDLTimer stopAndDestroyTimer

### ISSUE THREAD

**Issue by rosscintelematics** [smartdevicelink/sdl_ios#1179]
### Bug Report
App crashes at `[SDLTimer stopAndDestroyTimer] + 92`. There isn't a whole lot more information on this other than it was observed approximately 25 times over one day before being rolled back. I have attached an example crash report. Raising this for tracking purposes, fully aware that more information would be needed to properly investigate.

[2131_crash_.txt](https://github.com/smartdevicelink/sdl_ios/files/2943791/com.aaa.auto-club_issue_2131_crash_.txt)

```
#16. Crashed: com.sdl.protocol.receive
0  Foundation                     0x19a332601 -[NSCFTimer fire] + 65
1  SmartDeviceLink                0x1083493fc -[SDLTimer stopAndDestroyTimer] + 92
2  SmartDeviceLink                0x108349254 -[SDLTimer cancel] + 32
3  SmartDeviceLink                0x10832c9e0 -[SDLProxy handleProtocolStartSessionACK:] + 92
4  SmartDeviceLink                0x10834ec78 -[SDLProtocol handleProtocolStartServiceACKMessage:] + 1396
5  SmartDeviceLink                0x108371708 -[SDLProtocolReceivedMessageRouter sdl_dispatchControlMessage:] + 792
6  SmartDeviceLink                0x10837132c -[SDLProtocolReceivedMessageRouter handleReceivedMessage:] + 148
7  SmartDeviceLink                0x10834e6a4 __30-[SDLProtocol processMessages]_block_invoke + 60
8  libdispatch.dylib              0x1992b56c8 _dispatch_call_block_and_release + 24
9  libdispatch.dylib              0x1992b6484 _dispatch_client_callout + 16
10 libdispatch.dylib              0x19925dbd0 _dispatch_lane_serial_drain$VARIANT$mp + 592
11 libdispatch.dylib              0x19925e718 _dispatch_lane_invoke$VARIANT$mp + 432
12 libdispatch.dylib              0x199266eb8 _dispatch_workloop_worker_thread + 600
13 libsystem_pthread.dylib        0x1994990dc _pthread_wqthread + 312
14 libsystem_pthread.dylib        0x19949bcec start_wqthread + 4
```

##### Reproduction Steps
There are no reproduction steps at this time. Crash was submitted via Crashlytics in production.

##### Expected Behavior
App does not crash.

##### Observed Behavior
App crashes

##### OS & Version Information
* iOS Version: iPhone 7 on 12.1.4
* SDL iOS Version: 4.7.5
* Testing Against: Unknown

##### Test Case, Sample Code, and / or Example App
As above, there isn't more information on this other than it happened.

**Comment by NicoleYarroch** [comment]
@rosscintelematics What version of SDL_iOS did you roll back to? Was it 4.7.4?

**Comment by rosscintelematics** [comment]
Hey @NicoleYarroch, this deployment was the first launch with the SDL app present, so the rollback was to a version of the app without SDL at all.

**Comment by joeljfischer** [comment]
The symbolicated crash logs help and we'll look into it, but this is a version of SDL from 2017, and I would recommend using an updated version.

**Comment by theresalech** [comment]
This issue was fixed in the release of SDL iOS 5.0.  If you can update your app to 5.0, you should see the issue as resolved.
